bin/run-suite

```diff
@@ -147,6 +153,7 @@ if __name__ == '__main__':
     for filename in files:
       benchmark_name = filename
       benchmark_executable = os.path.realpath(filename)
+      print install_dir, benchmark_name, benchmark_executable, filename
```

Comment:
Please use python3 syntax ;) `print(...)`.
If you execute the script with python 2, this might be an explanation why some things such as the detection whether the script executes itself don't work correctly.

Reply:
I don't think it was a python version problem. The script tries to run itself if you use a different directory than sycl-bench/bin to install benchmark binaries. Try it.

Reply:
I've just made a run with python2 and it worked - so that's not the culprit. Not sure what you mean - I have my installed script and benchmark in `$ROOT/cmake-build/install/bin` and it works fine. Can you please describe how your setup looks like? Where do you install to?

Reply:
Try by specifying -DCMAKE_INSTALL_PREFIX=build-dir during cmake.

Reply:
Tried this, still working ;) Maybe there is a symlink or something like that in your path? Anyway, having thought about it I now believe just moving the benchmarks to a subdirectory is the most elegant solution.